controllers/goflowkube/goflowkube_objects.go

```diff
@@ -20,6 +22,9 @@ const configVolume = "config-volume"
 const configPath = "/etc/goflow-kube"
 const configFile = "config.yaml"
 
+// annotation name to facilitate pod restart after any external configuration change
+const podConfigurationDigest = "goflow_kube_config_digest"
```

Comment:
Should we use some kind of qualified name for the annotation? e.g. `netobserv.openshift.org/goflow_kube_config_digest`

Reply:
yes actually you could reuse the domain from the CRD: `flows.netobserv.io` or just `netobserv.io`